src/plugins/data/public/ui/index.ts

```diff
@@ -21,8 +21,8 @@ export { SuggestionsComponent } from './typeahead/suggestions_component';
 export { IndexPatternSelect } from './index_pattern_select';
 export { FilterBar } from './filter_bar';
 export { QueryStringInput } from './query_string_input/query_string_input';
+export { SearchBar, SearchBarOwnProps, SearchBarProps } from './search_bar';
```

Comment:
Should `SearchBarOwnProps` be exposed externally or can it be removed? AFAICT it is just an implementation detail.

I ask because currently, in `src/plugins/data/public/index.ts`, we do a `export * from './ui';` so all of this is exposed publicly.

Reply:
Smaller API! Done!